nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/indexing/conditions/EqualsCondition.java

```diff
@@ -43,7 +43,7 @@ public EqualsCondition(Number value) {
      */
     @Override
     public int condtionNum() {
-        return 10;
+        return 0;
```

Comment:
Why is this 0 now?

Reply:
@treo this was due to the conditions. See new javadoc: https://github.com/eclipse/deeplearning4j/blob/efbeccf3985d0cc1bfa0a012c16cae29e1075a76/nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/indexing/conditions/Condition.java#L36 here basically we could put equals (0) or absolute difference equals. I'm ok with either 0 or 10 here.